frontend/src/services/effects.ts

```diff
@@ -104,6 +104,11 @@ export function withEffects(store: Store) {
       // Reset adequacies.
       store.set('adequacies')({})
 
+      // If there is no providers or no representative points, return early.
```

Comment:
`If there are...`

also what does it mean to return early from the user's perspective?